src/Domain/CoordinateMaps/Wedge3D.cpp

```diff
@@ -0,0 +1,328 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/CoordinateMaps/Wedge3D.hpp"
+
+#include "DataStructures/MakeWithValue.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "ErrorHandling/Assert.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+
+namespace CoordinateMaps {
+
+Wedge3D::Wedge3D(const double inner_radius, const double outer_radius,
+                 const Direction<3> positioning_of_wedge,
+                 const double inner_sphericity)
+    : inner_radius_(inner_radius),
+      outer_radius_(outer_radius),
+      positioning_of_wedge_(positioning_of_wedge),
+      inner_sphericity_(inner_sphericity) {
+  ASSERT(inner_radius > 0, "The inner radius must be greater than zero.");
+  ASSERT(outer_radius > inner_radius,
+         "The outer radius must be larger than the inner radius.");
+  ASSERT(inner_sphericity >= 0 and inner_sphericity <= 1,
+         "Inner sphericity must be between 0 and 1");
+}
+
+template <typename T>
+std::array<std::decay_t<tt::remove_reference_wrapper_t<T>>, 3> Wedge3D::
+operator()(const std::array<T, 3>& x) const {
+  using ReturnType = std::decay_t<tt::remove_reference_wrapper_t<T>>;
+
+  const ReturnType xi = x[0];
+  const ReturnType eta = x[1];
+  const ReturnType zeta = x[2];
+  ReturnType x_inner_cubical_face = xi * inner_radius_ / sqrt(3.0);
+  ReturnType y_inner_cubical_face = eta * inner_radius_ / sqrt(3.0);
+  ReturnType z_inner_cubical_face =
+      make_with_value<ReturnType>(xi, inner_radius_ / sqrt(3.0));
+  const ReturnType x_unit_spherical_face = tan(M_PI_4 * xi);
```

Comment:
note to self: variables of the form `tan(M_PI_4*name)` will be named `equiangular_name` and will be called so in the documentation